app/lib/features/chat/widgets/custom_message_builder.dart

```diff
@@ -43,12 +43,18 @@ class CustomMessageBuilder extends StatelessWidget {
         return text == null
             ? const SizedBox.shrink()
             : Container(
-                width: sqrt(text.length) * 38.5,
-                padding: const EdgeInsets.all(18),
-                constraints: const BoxConstraints(minWidth: 57),
-                child: Text(
-                  text,
-                  style: Theme.of(context).textTheme.bodySmall,
+                padding: const EdgeInsets.only(left: 10, bottom: 5),
+                child: RichText(
+                  text: TextSpan(
+                    text: message.author.id,
+                    style: Theme.of(context).textTheme.bodySmall,
+                    children: [
+                      TextSpan(
+                        text: '\t$text',
```

Comment:
`\t`? Shouldn't we just give it an internal padding on the left?

Reply:
I suppose it meant here for applying space between different text spans.